src/Smart.CFA.Pages/Pages/Error.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@page
```

Comment:
Could you create an issue on the `smartcoop/design` project to ask for such a page ?

Reply:
It's auto-generated by the template of a Razor Page webapp. Most of the files of the project will disappear (but I needed a quick template to test the flow)

Reply:
Yeah, but I guess this makes sense to have some error pages using the design system ?

Reply:
Definitely and having a 404 too.